flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
you could just fix the broken code instead.

Reply:
Code should be 

```
bool is_ascii(const char str[])
{
    for (int i = 0; str[i] != '\0'; i++) {
        if (str[i] > 127) {
            return false;
        }
    }
    return true;
}
```

right? How do I patch that though, can you supply patch even if it's meson

Reply:
Apparently author of the package marked it as x86_64 arch only

[aur pkgbuild](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wlrctl)

Reply:
that will not build on x86_64.

Reply:
> Apparently author of the package marked it as x86_64 arch only
> 
> [aur pkgbuild](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wlrctl)

then don't build this on aarch64.

Reply:
and by this I mean wlrctl.